spec/models/user_spec.rb

```diff
@@ -0,0 +1,36 @@
+require "rails_helper"
+
+RSpec.describe User do
+  let!(:user) { User.create(name: "Markus",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.